release-notes.md

```diff
@@ -1,3 +1,74 @@
+# Quasar-17.17.0
+## Add EmailEventNotificationCoEditorsChangedEmailNewCoEditors template ##
```

Comment:
put all the relevant scripts under one section of co-editor/publishers event notification changes and distinguish between the add case and the upload case. if already exist in the env no need to run add